mautic/mautic
# Custom contacts columns

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | This PR closes #120 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added option custom  contact's columns from configurations.

![image](https://user-images.githubusercontent.com/462477/57571459-e9a85c80-740e-11e9-81c0-d4e17d3d218d.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7511)
2. You should see default columns (like before)
3. Go to Configurations > Contact Settings tag and see new Contact List Settings part
4. Change some columns  and save
5. Go to contacts and see If your changes was applied

![image](https://user-images.githubusercontent.com/462477/57571515-a4385f00-740f-11e9-99da-69e019e8800c.png)
